test/unit/gen/certificate.go

```diff
@@ -172,7 +172,7 @@ func SetCertificateNotBefore(p metav1.Time) CertificateModifier {
 	}
 }
 
-func SetCertificateRenewalTIme(p metav1.Time) CertificateModifier {
```

Comment:
Fixing this typo caused a build failure in `cmd/ctl/pkg/status/certificate/certificate_test.go:238` Could you also update it there?

Reply:
Good catch, looks like I actually had changed it but missed committing it. Thats why the tests were passing locally.